File: src/main/java/com/sm/carwashmonitor/dto/StatisticsSummaryDTO.java
package com.sm.carwashmonitor.dto;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class StatisticsSummaryDTO {
    List<StationStatisticsDTO> allStationStatistics;
    Float totalWaterCost;
    Float totalDetergentCost;
    Float totalWaxCost;
    Float totalRevenue;
}

File: Dtos/StatisticsSummaryDto.cs
namespace CarwashMonitor.Dtos;

public class StatisticsSummaryDto
{
    public List<StationStatisticsDto>? allStationStatistics = [];
    public float TotalDetergentCost;
    public float TotalRevenue;
    public float TotalWaterCost;
    public float TotalWaxCost;
}